test/e2e/common.go

```diff
@@ -127,14 +127,23 @@ func (test *e2eTest) DeleteTestNamespace(t *testing.T, namespace string) {
 // WaitForNamespaceDeleted wait until namespace is deleted
 func (test *e2eTest) WaitForNamespaceDeleted(t *testing.T, namespace string) {
 	logger := Logger{}
-	deleted := checkNamespaceDeleted(t, namespace, MaxRetries, logger)
+	deleted := checkNamespace(t, namespace, false, MaxRetries, logger)
 	if !deleted {
 		t.Fatalf("Error deleting namespace %s, timed out", namespace)
 	}
 }
 
+// WaitForNamespaceCreated wait until namespace is created
+func (test *e2eTest) WaitForNamespaceCreated(t *testing.T, namespace string) {
+	logger := Logger{}
+	created := checkNamespace(t, namespace, true, MaxRetries, logger)
+	if !created {
+		t.Fatalf("Error creating namespace %s, timed out", namespace)
```

Comment:
%q so it's quoted. I think it would be better to return an error and print that instead of assuming it's due to a timeout. Then see next comment below....

Reply:
```golang
t.Fatalf("Error creating namespace %s, timed out", namespace)
```
Fails the test and shows message to user.